File: CPP/Kode Program/Obat.cpp
#include <iostream>
#include <vector>

using namespace std;

class Obat{
    private:
        //Atribut Class Obat
        string namaObat;
        int harga;
        string bentuk;
    
    public:
        //Constructor
        Obat(string namaObat, int harga, string bentuk){
            this->namaObat = namaObat;
            this->harga = harga;
            this->bentuk = bentuk;
        }

        //Getter dan Setter
        string getNamaObat(){
            return this->namaObat;
        }

        void setNamaObat(string namaObat){
            this->namaObat = namaObat;
        }

        int getHarga(){
            return this->harga;
        }

        void setHarga(int harga){
            this->harga = harga;
        }

        string getBentuk(){
            return this->bentuk;
        }

        void setBentuk(string bentuk){
            this->bentuk = bentuk;
        }

        //Cetak info / data dari sebuah obat
        void printObat(){
            cout << "Nama Obat : " << getNamaObat() 
            << ", Harga : " << getHarga() 
            << ", Bentuk : " << getBentuk() << endl;
        }

        //Destructor
        ~Obat(){
        }
};

File: Python/Kode Program/Obat.py
class Obat:
    def __init__(self, namaObat, harga, bentuk):
        self.namaObat = namaObat
        self.harga = harga
        self.bentuk = bentuk

    # Getter & Setter
    def getNamaObat(self):
        return self.namaObat

    def setNamaObat(self, namaObat):
        self.namaObat = namaObat

    def getHarga(self):
        return self.harga

    def setHarga(self, harga):
        self.harga = harga

    def getBentuk(self):
        return self.bentuk

    def setBentuk(self, bentuk):
        self.bentuk = bentuk

    # Print info obat
    def printObat(self):
        print(f"Nama Obat : {self.getNamaObat()}, "
              f"Harga : {self.getHarga()}, "
              f"Bentuk : {self.getBentuk()}")
